NoSuchMethodException: liquibase.change.AddColumnConfig.<init>() when running native image

### Expected Behavior

The program should execute all migrations correctly.

### Actual Behaviour

Following exception is thrown: `Caused by: java.lang.NoSuchMethodException: liquibase.change.AddColumnConfig.<init>()`

### Steps To Reproduce

Not sure, I have a set of ordinary changesets, which work when using the jvm version. 

probably this class `liquibase.change.AddColumnConfig.<init>()` needs to be added to the reflection-config? 

### Environment Information

Building a native image. 

### Example Application

_No response_

### Version

3.2.0

--- a/liquibase/src/main/resources/META-INF/native-image/io.micronaut.liquibase/micronaut-liquibase/reflect-config.json
+++ b/liquibase/src/main/resources/META-INF/native-image/io.micronaut.liquibase/micronaut-liquibase/reflect-config.json
@@ -32,6 +32,17 @@
     "allPublicMethods":true
   },
   {
+    "name": "liquibase.change.AddColumnConfig",
+    "allDeclaredFields": true,
+    "allPublicMethods":true,
+    "methods": [
+      {
+        "name": "<init>",
+        "parameterTypes": []
+      }
+    ]
+  },
+  {
     "name":"liquibase.change.ChangeFactory",
     "methods":[{"name":"<init>","parameterTypes":[] }]
   },

